homeassistant/components/homekit_controller/cover.py

```diff
@@ -262,33 +265,39 @@ async def async_set_cover_position(self, **kwargs: Any) -> None:
             {CharacteristicsTypes.POSITION_TARGET: position}
         )
 
+    def _getArcTiltPosition(
```

Comment:
Same comment about camel case for function and variable names. 

And this doesn't use self, so lift it out of the class and drop the self argument.

Reply:
moved